main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    data = json.load(open("players.json", "r")).items()
+
+    for player_name, player_data in data:
+        race_data = player_data["race"]
+        guild_data = player_data["guild"]
+
+        race = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data["description"]}
+        )[0]
```

Comment:
Please, use unpacking instead of retrieving value by index here and in the other places as well